ortho_seq_code/testdata/protein_test_data/CDR3H_16_seqs.csv

```diff
@@ -0,0 +1,8 @@
+ARSYDILTGYRDAFDI
```

Comment:
cool! good job agetting the test data.

use https://github.com/czbiohub/sencha/blob/master/tests/test_translate.py#L256 something like this as starting point for tests and try to develop tests as well